spdx-utils/src/test/kotlin/SpdxExpressionTest.kt

```diff
@@ -321,5 +322,19 @@ class SpdxExpressionTest : WordSpec() {
                 spdxExpression.disjunctiveNormalForm() shouldBe dnf
             }
         }
+
+        "valid choices" should {
+            "list the valid choices for a complex expression" {
+                val spdxExpression = "(a OR b) AND (c AND (d OR e))".parse()
+                val choices = listOf(
+                    "a AND (c AND d)",
```

Comment:
Would `a AND c AND d` or ratjer `(a AND c) AND d` be a valid choice according to the valid choices return value?
Note: I'm aware these are equivalent but depending on how you're going to use this function it may be an issue.

How about making the test independent of the ordering? or is it already?

Reply:
At this point the order is important and the parenthesis are required, but this changes later in this PR with the new implementation of `equals`.